src/components/users/EditProfileButton.tsx

```diff
@@ -1,14 +1,48 @@
+import React from 'react'
 import { Button, ButtonVariant } from '../ui/BaseButton'
-
 import forOwnerOnly from '../../js/auth/forOwnerOnly'
 import { WithOwnerProfile } from '../../js/types/User'
+import Tooltip from '../ui/Tooltip'
 
 interface EditProfileButtonProps extends WithOwnerProfile {
+  loginsCount?: number // Add loginsCount prop
 }
 
-function EditProfileButton (): JSX.Element {
+/**
+ * EditProfileButton component displays a "Edit" button with optional tooltip based on loginsCount prop.
+ * @param {EditProfileButtonProps} loginsCount - The number of times a user has logged in since account creation.
+ * @returns {JSX.Element} - JSX element representing the EditProfileButton component.
+ */
+function EditProfileButton ({ loginsCount }: EditProfileButtonProps): JSX.Element {
   return (
-    <Button href='/account/edit' label='Edit' variant={ButtonVariant.OUTLINED_DEFAULT} size='sm' />
+    <>
+      {loginsCount === 1
```

Comment:
what if the user ignores the tooltip and closes the browser?  The next time will they see the reminder?

Reply:
You're right, I didn't really consider that. I think some sort of formalized onboarding process would be beneficial.